server/test/services/export/ProgramJsonSamplerTest.java

```diff
@@ -23,28 +31,59 @@ public class ProgramJsonSamplerTest extends ResetPostgres {
 
   private ProgramJsonSampler programJsonSampler;
 
-  /**
-   * We need to mock the AutoValue in order to mock on streamQuestionDefinitions(), and this is
-   * substantially simpler than seeding the complex ProgramDefinition AutoValue with data.
-   */
-  @SuppressWarnings("DoNotMockAutoValue")
-  @Mock
-  private ProgramDefinition mockProgramDefinition;
```

Comment:
Since we need a couple more pieces of data from the test `ProgramDefinition`, let's stop mocking it and actually create one, for better test fidelity.

Reply:
Cool!